gtk-2.0/gtkrc

```diff
@@ -168,8 +168,8 @@ style "murrine-thin" {
 # Notebook
 
 style "clearlooks-notebook-bg" {
-	bg[NORMAL] = @bg_color
-	bg[ACTIVE] = shade (0.80, @bg_color)
+	bg[NORMAL] = @base_color
+	bg[ACTIVE] = shade (0.87, @base_color)
```

Comment:
could you please do that for both gtk versions once when you'll be rebasing (i think after we'll clarify about svg-s)?